test/correctness/vector_cast.cpp

```diff
@@ -19,10 +19,19 @@ DECL_SOT(int32_t);
 DECL_SOT(float);
 DECL_SOT(double);
 
+template <typename T>
+bool is_type_supported(int vec_width, const Target &target) {
+    return true;
```

Comment:
Looking at Target::supports_type, I think that the implementation of that is wrong, because it returns types are not supported if any GPU feature is enabled, even if the code isn't running on that device. I vaguely remember discussing this and we decided that was OK... but that seems crazy now?